.travis.yml

```diff
@@ -8,8 +8,10 @@ install:
    - pip install pytest
    - pip install pytest-cov
    - pip install coveralls
+   - npm install -g markdownlint-cli
```

Comment:
does this mean we will need to add node.js to install?

Reply:
yes, node.js is prerequisite for npm